src/components/checkbox/checkbox.ts

```diff
@@ -8,10 +8,9 @@ import {
     Provider,
     Renderer,
     ViewEncapsulation,
-    forwardRef,
-    AfterContentInit
-} from '@angular/core';
-import {NG_VALUE_ACCESSOR, ControlValueAccessor} from '@angular/common';
+    forwardRef
+} from '../../core/core';
```

Comment:
This was just commited for testing purposes and I already force-pushed to remove it :)